### ISSUE TITLE
Multiple select

### ISSUE THREAD

**Issue by multiOTP** [wffranco/vue-strap#105]
Hello
in the documentation we found this
`<v-select v-model="select.value" :options="select.options" options-value="val" multiple name="animals[]" limit="3" placeholder="Using placeholder" search justified required disabled clear-button close-on-select></v-select>`

But we had to replace `limit="3"` by `:limit=3`

Can you update the documentation or tells us what we did wrong ?
Thanks, have a nice day

**Comment by pramjeet** [comment]
`limit="3"` gives a string but the later one gives an integer, I think that's why it was causing issues.

**Comment by wffranco** [comment]
yeah you right, a little mistake because in vue2 was removed coerse, in vue1 this prop auto convert string to number.
